code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -394,6 +394,12 @@
 	var/fortify = 0
 	var/crest_defense = 0
 
+	// Baneling vars
+	/// How long smoke lasts for when baneling explodes
+	var/smoke_duration = 4
```

Comment:
Better var name, there's half a dozen xeno smokes this could refer to at a glance.